src/lib/client.js

```diff
@@ -103,7 +103,7 @@ export default class Client {
       return reject('charges should be a charges object');
     }
 
-    const url = pdpHost(marketingUI, this.platform) + '/checkout.html?' + this.key + '&charges=' + [charges.data.total, charges.data.shipping, charges.data.tax, charges.data.discount_amount, charges.data.grand_total].join(',');
+    const url = pdpHost(marketingUI, this.platform) + '/checkout.html?public_key=' + this.key + '&charges=' + [charges.data.total, charges.data.shipping, charges.data.tax, charges.data.discount_amount, charges.data.grand_total].join(',');
```

Comment:
@cjdeg same problem here with the unnamed url param, though probably far less severe since I don't think this method gets used all that much.